src/app/controllers/payment/payment.js

```diff
@@ -0,0 +1,45 @@
+angular.module('crowdFundApp').controller('payment',
+  function($scope, $http, CONFIG, PaymentService, toaster, $stateParams, CampaignService) {
+
+    CampaignService.getCampaign().then(
+      function(success) {
+        $scope.campaign = success.data
+      }
+    )
+    // Stripe Response Handler
+    Stripe.setPublishableKey('pk_test_xTQpXGarHpdw1K62JcYxZ1ll')
```

Comment:
Please don't push any secret keys into the repo, and use `Environment Variables` instead!